src/main/java/training/commands/Elbow/UpElbowCommand.java

```diff
@@ -0,0 +1,36 @@
+package training.commands.Elbow;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import training.subsystems.Elbow.Elbow;
+
+public class UpElbowCommand extends Command {
+    private final Elbow elbowSubsystem;
+    private final Rotation2d addedAngle;
+    private final Rotation2d originalAngle;
+
+    public UpElbowCommand(Rotation2d angle) {
+        elbowSubsystem = Elbow.getInstance();
+        addRequirements(elbowSubsystem);
+        addedAngle = angle;
+        originalAngle = elbowSubsystem.getMotorAngle();
+    }
+
+    public UpElbowCommand() {
+        super();
+        elbowSubsystem = Elbow.getInstance();
+        addRequirements(elbowSubsystem);
+        addedAngle = Rotation2d.fromDegrees(Constants.DEFAULT_LIFT_DEG);
+        originalAngle = elbowSubsystem.getMotorAngle();
```

Comment:
i meant use the constructor above instead of duplicating code. Same in all relevant places